custom/icds_reports/tasks.py

```diff
@@ -1743,22 +1743,27 @@ def get_data_not_in_ucr(status_record):
     chunk_size = 1000
     for chunk in chunked(matching_records_for_db, chunk_size):
         doc_ids = [val[0] for val in chunk]
-        docs_in_ucr = _get_docs_in_ucr(domain, status_record.table_id, doc_ids)
+        doc_id_and_inserted_in_ucr = _get_docs_in_ucr(domain, status_record.table_id, doc_ids)
```

Comment:
merging this now so we can get it out but can you change this name back. I think the new name is much more confusing that the old.